src/Illuminate/Foundation/Console/Tinker/Presenters/IlluminateApplicationPresenter.php

```diff
@@ -0,0 +1,72 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ObjectPresenter;
+use Illuminate\Foundation\Application;
+
+class IlluminateApplicationPresenter extends ObjectPresenter {
+
+	/**
+	 * Illuminate Application methods to include in the presenter.
+	 *
+	 * @var array
+	 */
+	protected static $appProperties = [
```

Comment:
This is going to get easily outdated and forgotten about when we methods are added. How can we make sure that doesn't happen?